src/CountUp.tsx

```diff
@@ -1,9 +1,13 @@
 import PropTypes from 'prop-types';
+import { CountUp } from 'countup.js';
 import React, { Component } from 'react';
 import warning from 'warning';
 import { createCountUpInstance } from './common';
+import { CountUpProps } from '../index';
 
-class CountUp extends Component {
+const AVAILABLE_TAGS = ['input', 'text', 'span', 'div', 'p'];
```

Comment:
How do we deal with  other tags such as section, button, etc.?

Reply:
@glennreyes to be honest idk how to do it nicely. I can add this tags here, but itsn,t perfect

Reply:
Ideally we don't maintain a list of html tags. How does this solve better than the previous condition? Are they equivalent?

```js
          (this.containerRef.current instanceof HTMLElement ||
            this.containerRef.current instanceof SVGTextElement ||
            this.containerRef.current instanceof SVGTSpanElement)
```

Reply:
@glennreyes I changed it to make TS happy cause with this condition I got error:
```
TS2358: The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.
```

It's ok with just 
```js
this.containerRef.current instanceof HTMLElement
```

but I'm not sure that is right

Reply:
Maybe we can cast with `any`? So:
```ts
this.containerRef.current as any instanceof ...
```

Reply:
@glennreyes some of changes broke this checks for svg elements :(
I just got ```ReferenceError: SVGTextElement is not defined``` and I can't figure with it. Rewrite it like so:
```js
const AVAILABLE_SVG_TAGS = ['text', 'tspan'];

      warning(
        this.containerRef.current instanceof HTMLElement ||
          !!(
            this.containerRef.current &&
            AVAILABLE_SVG_TAGS.includes(
              this.containerRef.current!.tagName.toLowerCase(),
            )
          ),
        `Couldn't find attached element to hook the CountUp instance into! Try to attach "containerRef" from the render prop to a an HTMLElement, eg. <span ref={containerRef} />.`,
      );
```